### PR TITLE
fix(curriculum): external passport-GitHub's link in QA

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

clear the 6th task in #40749

### CODE REVIEW

File: curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/implementation-of-social-authentication-ii.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ dashedName: implementation-of-social-authentication-ii
...
+To set up the GitHub strategy, you have to tell Passport to use an instantiated `GitHubStrategy`, which accepts 2 arguments: an object (containing `clientID`, `clientSecret`, and `callbackURL`) and a function to be called when a user is successfully authenticated, which will determine if the user is new and what fields to save initially in the user's database object. This is common across many strategies, but some may require more information as outlined in that specific strategy's GitHub README. For example, Google requires a *scope* as well which determines what kind of information your request is asking to be returned and asks the user to approve such access.
+
+The current strategy we are implementing authenticates users using a GitHub account and OAuth 2.0 tokens. The client ID and secret obtained when creating an application are supplied as options when creating the strategy. The strategy also requires a `verify` callback, which receives the access token and optional refresh token, as well as profile which contains the authenticated user's GitHub profile. The `verify` callback must call `cb` providing a user to complete authentication.
```
Comment (lasjorg) [inline curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/implementation-of-social-authentication-ii.md:15]:
```suggestion
The current strategy we are implementing authenticates users using a GitHub account and OAuth 2.0 tokens. The client ID and secret obtained when creating an application are supplied as options when creating the strategy. The strategy also requires a `verify` callback, which receives the access token and optional refresh token, as well as `profile` which contains the authenticated user's GitHub profile. The `verify` callback must call `cb` providing a user to complete authentication.
```

Looking at the docs this is taken from, `profile` should be in a code block.

**Review by lasjorg — CHANGES_REQUESTED** [review]
A small fix is needed.

File: curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/implementation-of-social-authentication-ii.md (hunk 1 above)
Comment (Sboonny) [reply curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/implementation-of-social-authentication-ii.md]: 👍

Comment (lasjorg) [reply curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/implementation-of-social-authentication-ii.md]: dub

**Comment by lasjorg** [comment]
Not sure what happened there. Sorry.

**Comment by Sboonny** [comment]
No worries happened to me yesterday as well

**Review by lasjorg — APPROVED** [review]
LGTM.

I know the one test is failing but it seems to be a known issue.

**Review by raisedadead — APPROVED** [review]

**Pull request merged**